plans/001-fix-partial-index-upserts.md

```
@@ -0,0 +1,223 @@
+    inbox_id: input.inboxId,
+    created_by_profile_id: input.actorProfileId,
+  })
+  .eq("tenant_id", input.tenantId)
+  .eq("scope", input.scope)
+  .eq("match_value", matchValue)
+  .eq("is_active", true)
+  .select("id")
+  .single();
+```
+
+3. Any other error (or missing row after both attempts) → keep throwing
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/1/1

(MD029, ol-prefix)